src/Factory/AwsFactory.php

```diff
@@ -16,18 +15,19 @@ class AwsFactory implements FactoryInterface
 {
     /**
      * @param ContainerInterface $container
-     * @param string             $requestedName
-     * @param array|null         $options
+     * @param string $requestedName
+     * @param array|null $options
      * @return AwsSdk
      */
     public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
     {
         // Instantiate the AWS SDK for PHP
         $config = $container->get('Config');
         $config = isset($config['aws']) ? $config['aws'] : [];
+
         $config += [
             'ua_append' => [
-                'ZF2/' . Version::VERSION,
+                'ZF2/' . Module::VERSION,
```

Comment:
@cjyclaire 
Zend\Version is provided by a module called "zendfrawework/zend-version". This module has  a dependency on  "zendfrawework/zend-json" (~2.5).  On another hand Zend Framework 3 uses newer version of ""zendfrawework/zend-json" (3.0.0) Whenever someone is trying to use our module with Zend Framework 3 they will not be able to install it because of this conflict (composer will complain and force you to select one of the versions). That was the reason I suggested getting rid of the Zend\Version dependency or going around it in some fashion. :)